recipes/grepq/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set version = "1.4.1" %}
+{% set name = "grepq" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Rbfinch/{{ name }}/archive/refs/tags/{{ name }}-v{{ version }}.tar.gz
+  sha256: 6eb4be2403330acf185ac0249b27f384343b3184a34fc3c41af6357f61d18353
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('grepq', max_pin="x") }}
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
```

This **might** workaround the `ISO C99 and later do not support implicit function declarations` error you're getting on OSX. I don't have a machine with OSX, so I can't test locally, but I'd try this if you're stuck.
```suggestion
    - export CFLAGS="${CFLAGS} -Wno-implicit-function-declaration"
    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
```